pkg/util/cluster/cluster.go

```diff
@@ -403,38 +430,33 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 }
 
 func (c *Cluster) fixupNSGs(ctx context.Context, vnetResourceGroup, clusterName string) error {
```

Comment:
fixupNSGs should be left as is and new function created for the peering.

Reply:
done